fusion_toolbox/reaction_rate/bosch_hale.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+class Bosch_Hale_reactivity:
```

Comment:
```suggestion
class BoschHaleReactivity:
```
typically classes are camel case

Reply:
Fixed with https://github.com/tmarkj/fusion-toolbox/commits/main